code/modules/library/lib_items.dm

```diff
@@ -283,6 +284,67 @@
 	else
 		return ..()
 
+/obj/item/weapon/book/mindbook
+	name = "A Harmless Book"
+	icon = 'icons/obj/library.dmi'
+	icon_state ="book"
+	unique = 1
+	forbidden = 1
+	var/hallucinatory_cooldown
+
+/obj/item/weapon/book/mindbook/attack_self(mob/living/H)
+	if(ishuman(H))
+		var/mob/living/carbon/human/P = H
+		if((P.mind && P.mind.assigned_role == "Psychiatrist") || prob(50))
+			if(hallucinatory_cooldown > world.time)
+				to_chat(P, "The book needs to recharge.")
+				return
+			else
+				var/list/victim_list = list()
+				for(var/mob/living/carbon/human/V in view(P.client.view, P))
+					if(V.stat == DEAD)
+						continue
+					if(!(V.client && V.mind))
+						continue
+					if(PSY_RESIST in V.mutations)
+						continue
+					victim_list += V
+				if(!victim_list.len)
+					to_chat(P, "There are no available victims around!")
+					return
+				else
+					var/mob/living/victim = input("Select your hapless victim!", "Cancel") as null|mob in victim_list

```

Comment:
Technicallyyyy, this means you can select someone if they're in view, walk to the clear opposite side of the station while this menu is up, then finally click their name and it'll hit them with this.